src/Manager_TEST.cc

```diff
@@ -81,6 +119,73 @@ TEST(Ignition_TEST, RunLs)
   EXPECT_TRUE(mgr.RunConfig(config));
 }
 
+
+/////////////////////////////////////////////////
+TEST(Ignition_TEST, IGN_UTILS_TEST_DISABLED_ON_WIN32(RunEnvPre))
+{
+  // Test that environment is applied regardless of order
+  std::string testPath = "/tmp/ign-launch-env-test-pre";
```

Comment:
Will this work on Windows?

Reply:
No, I don't think this will work on Windows, I went ahead and marked it disabled with the macro.

I didn't think that we were up to launch on Windows yet, maybe something to figure out when we get there?